docs/source/tutorials/segmentation.ipynb

```diff
@@ -449,13 +449,13 @@
    "source": [
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "top_2_issues = np.argsort(-np.sum(issues, axis=(1, 2)))[:2]\n",
-    "assert (top_2_issues == [131,  29]).all()\n",
+    "assert (top_2_issues == [1,  21]).all()\n",
     "\n",
     "top_3_class_issues = np.argsort(-np.sum(class_issues, axis=(1, 2)))[:3]\n",
-    "assert (top_3_class_issues == [134,  66, 109]).all()\n",
+    "assert (top_3_class_issues == [17,  19, 0]).all()\n",
     "\n",
-    "highlighted_indices = [ 78,  16, 120,  34, 137,  51,  71,   8, 106,  94]\n",
-    "if not all(np.argsort(issues_from_score.sum((1,2)))[:10]==highlighted_indices):\n",
+    "highlighted_indices = [ 1,  21, 2,  24, 4, 3, 12]\n",
+    "if not all(np.argsort(-issues_from_score.sum((1,2)))[:7]==highlighted_indices):\n",
```

Comment:
just curious: why do all the argsorts require negation before them?

Reply:
`issues_from_score.sum((1,2))` returns a sum of the number of issues per image. Therefore images with more pixel issues have higher scores. Argsort sorts an array from lower to highest value so we negate the array prior to sorting to get the top 7 images with the most issues.